### DISCUSSION TITLE
2 scene but one is deleted

### DISCUSSION THREAD

**Discussion by HYPE-STUDIOS-PHILIP** [mozilla/hubs#3838] [Spoke Questions]
Hi!

Made a scene a little while ago called 'Raadsalen', I made another save as Raadsalen_v002, but that just made 2 scene if you search for 'Raadsalen'. I deleted the one scene in projects, so det correct one is the only one I see, but you can still find the 2 scenes if you search for them. Is this a update issue? or how do i remove the scene that I deleted in my prosjects?

Thanks!

Philip Schøyen

**Comment by pattersonbl2** [comment]
are you still having issues with this ?
